states: 
 state_1:     
  weight_optimaltime: 20.0                    
  xy_goal_tolerance: 0.1             
  yaw_goal_tolerance: 0.1            
  weight_obstacle: 800               
  weight_inflation: 0.5               
  weight_acc_lim_x: 10               
  weight_max_vel_x: 10               
  weight_acc_lim_y: 6                
  weight_max_vel_y: 6                
  weight_acc_lim_theta: 25           
  weight_max_vel_theta: 25           
  acc_lim_x: 2                     
  acc_lim_y: 2                     
  max_vel_x: 2                       
  max_vel_y: 1                       
  max_vel_theta: 3                   
  acc_lim_theta: 3                   
  min_obstacle_dist: 0.28            
  inflation_dist: 0.3                
  min_turning_radius: 0.00           
  weight_kinematics_nh: 300           
  cost_scaling_factor_global: 15.0    
  inflation_radius_global: 0.25        
  robot_radius_global: 0.21            
  cost_scaling_factor_local: 15.0     
  inflation_radius_local: 0.23         
  robot_radius_local: 0.21            
 state_2:  
  weight_optimaltime: 20
  xy_goal_tolerance: 0.1
  yaw_goal_tolerance: 0.1
  weight_obstacle: 800
  weight_inflation: 0.5
  weight_acc_lim_x: 7
  weight_max_vel_x: 7
  weight_acc_lim_y: 2
  weight_max_vel_y: 2
  weight_acc_lim_theta: 15
  weight_max_vel_theta: 15
  acc_lim_x: 3
  acc_lim_y: 3
  max_vel_x: 3
  max_vel_y: 3
  max_vel_theta: 3
  acc_lim_theta: 3
  min_obstacle_dist: 0.28
  inflation_dist: 0.3
  min_turning_radius: 0.00
  weight_kinematics_nh: 300
  cost_scaling_factor_global: 15.0
  inflation_radius_global: 0.25
  robot_radius_global: 0.18
  cost_scaling_factor_local: 15.0
  inflation_radius_local: 0.23
  robot_radius_local: 0.18
 state_3:   
  weight_optimaltime: 30
  xy_goal_tolerance: 0.05
  yaw_goal_tolerance: 0.05
  weight_obstacle: 800
  weight_inflation: 0.5
  weight_acc_lim_x: 5
  weight_max_vel_x: 5
  weight_acc_lim_y: 3
  weight_max_vel_y: 3
  weight_acc_lim_theta: 15
  weight_max_vel_theta: 15
  acc_lim_x: 1.2
  acc_lim_y: 0.3
  max_vel_x: 3
  max_vel_y: 2
  max_vel_theta: 3
  acc_lim_theta: 3
  min_obstacle_dist: 0.27
  inflation_dist: 0.3
  min_turning_radius: 0.00
  weight_kinematics_nh: 400
  cost_scaling_factor_global: 15.0
  inflation_radius_global: 0.1
  robot_radius_global: 0.2
  cost_scaling_factor_local: 15.0
  inflation_radius_local: 0.1
  robot_radius_local: 0.2
 state_4:   
  weight_optimaltime: 50
  xy_goal_tolerance: 0.1
  yaw_goal_tolerance: 0.1
  weight_obstacle: 800
  weight_inflation: 0.5 
  weight_acc_lim_x: 5
  weight_max_vel_x: 5
  weight_acc_lim_y: 2
  weight_max_vel_y: 2
  weight_acc_lim_theta: 15
  weight_max_vel_theta: 15
  acc_lim_x: 3
  acc_lim_y: 2
  max_vel_x: 2
  max_vel_y: 2
  max_vel_theta: 2.2
  acc_lim_theta: 3.1
  min_obstacle_dist: 0.27
  inflation_dist: 0.3
  min_turning_radius: 0.00
  weight_kinematics_nh: 400
  cost_scaling_factor_global: 15.0
  inflation_radius_global: 0.21
  robot_radius_global: 0.2
  cost_scaling_factor_local: 15.0
  inflation_radius_local: 0.21
  robot_radius_local: 0.2
